spec/spec_helper.rb

```diff
@@ -56,6 +56,8 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.alias_it_should_behave_like_to :it_has_behavior, 'has behavior:'
+
```

Comment:
The conventional invocation for shared examples is simply `it_behaves_like`.  Why do we need something unconventional?